pkg/roachprod/errors/errors.go

```diff
@@ -149,6 +155,9 @@ func ClassifyCmdError(err error) Error {
 		if exitCode == 255 {
 			return NewSSHError(err)
 		}
+		if exitCode == 100 {
```

Comment:
Is the assumption here that we pretty much only see exit code 100 for apt failures, similar to the exit code == 255 for ssh flakes? Curious because I see before you had string matching for `apt`.

Reply:
Oops saw your other comment right after I posted this one 😄

Reply:
The chance of conflating `AptError` with some other command error doesn't seem overly concerning. If we notice an increase in the frequency of `AptError`, we can refine them.